mmcv/runner/hooks/logger/dvclive.py

```diff
@@ -11,48 +13,62 @@ class DvcliveLoggerHook(LoggerHook):
     It requires `dvclive`_ to be installed.
 
     Args:
-        path (str): Directory where dvclive will write TSV log files.
+        model_file (str):
```

Comment:
The purpose of this parameter is to save the latest checkpoint file, so isn't it possible to have a more intuitive name for it.

Reply:
It's a little tricky. 

The purpose of the parameter is to save the model at the end of each epoch. This is intended to be combined with a [DVC-specific feature](https://dvc.org/doc/user-guide/experiment-management/checkpoints)  called `checkpoints` (unfortunately a very reused term 😅). DVC expects MMCV (or other frameworks) to overwrite `model_file` at the end of each epoch.

It is necessary to use a specific string (it might not be the same name used for enabling MMCV checkpoints).

For users not using `DVC checkpoints`, they can just omit the parameter and use this integration just for logging (and the `MMCV Checkpoint` callback if they want to save the model)

Reply:
Got it.